packages/ui/src/memberships/components/MemberRoles.tsx

```diff
@@ -85,6 +96,32 @@ export const MemberRoles = ({ size, max, wrapable, roles }: MemberRolesProps) =>
   )
 }
 
+export interface MemberRolePopupContentProps {
+  roles: MemberRole[]
+}
+
+export const MemberRolePopupContent = ({ roles }: MemberRolePopupContentProps) => {
+  const groupAddress = `#/working-groups/${groupNameToURLParam(roles[0].groupName)}`
+  return (
+    <>
+      {roles.map((role, index) => (
+        <PopupRoleItem key={index}>
+          <PopupRoleTitle>
+            {memberRoleTitle(role)} {roles.length > 1 ? index + 1 : ''}
+          </PopupRoleTitle>
+          {role.createdAt && (
```

Comment:
How big the effort it would be to make `createdAt` obligatory?

Reply:
We have so called Member Role(default one) and it doesn't have time